01_wrangling/wrangle_all.R

```diff
@@ -248,6 +248,9 @@ df_ind_sev_data <- map_dfr(
 )
 
 df_ind_sev_data %>%
+  filter(
+    !is.na(severity) & severity > 0
```

Comment:
Before we approve, can you go through and confirm we are correctly deleting the countries here? Just provide a list in this thread of the countries being removed and why, such as the data is for mine action subsector in Colombia, Haiti for the migration sector, etc.